### PR TITLE
This assignment consists of making a clone of Apple's web page.

### PR DESCRIPTION
**Pull request by Wusinho**
This assignment consists of making a clone of Apple's web page.
In this part of the project we used:
- HTML5
* HTML5 semantic
- CSS3
* flex box
* css grid
* styleling
* Background-image
* Linear Gradient

### CODE REVIEW

**Review by Temmarie — CHANGES_REQUESTED** [review]
## Changes Required! :recycle: 
### Hello Team! 
 
### You have done well so far :clap: However, there are some changes you have to make for your project to be approved. Kindly work on them and submit them for another code review.


**General** 

- [ ] Your project looks great. However, your project cannot be approved until you write a descriptive PR Message. It's essential that your repository is professional, as this is one of the things that potential colleagues will look at when recruiting team members. Please refer to [pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/) for more information.

- [ ] Please fix your linter errors.


**Recommended**

- [ ] The gradient on the nav-bar looks good but it's a bit dark compared to the actual site. Try to make it look as similar as possible. 

 - [ ] The font-family used in the nav bar as well as throughout the entire page differs from that used in the web archive. To fix this issue, I'll recommend using a [Font Picker chrome extension](https://chrome.google.com/webstore/detail/font-picker/mmjbimgpcbaegjiieojddickpjbdkeej). 
In addition to that, I'll also advise using the SVG images used in the web archive instead of text, if you choose to do so, you can use [SVG Export chrome extension](https://chrome.google.com/webstore/detail/svg-export/naeaaedieihlkmdajjefioajbbdbdjgp?hl=en-GB).

- [ ] The four images at the base of the page is bigger than that of the actual site. Please use this to fix the images(a1 and it's subsequent selectors), and after that fix the issues that may arise. 
`a1 { 
       width: 364px;
       height: 200px;
}`

Note that if you add the style to the ".a1, .a2, .a3, .a4" style you have in your css , you'll get a horizontal overflow.

- [ ] The footer has a tiny little issue. The footer links should be centralized. I'll recommend putting them in a separate container so as to easily position them.



__Kindly make the necessary changes and submit another code review request.__
_Feel free to reach me on slack for any inquiries about this review @tamara._ :wink: 
__Grace Tamara Ekunola __
[GitHub](https://github.com/temmarie) | [LinkedIn](https://www.linkedin.com/in/ekunola-grace/) | [Twitter](https://twitter.com/TemmarieW)

File: README.md [hunk 1]
```diff
@@ -1 +1,51 @@
...
+
+- GitHub: [@Abdelrhman-Amin](https://github.com/AbdelrhmanAmin)
+- Twitter: [@Abdo-Amin](https://twitter.com/AbdoAmi60489112)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/abdo-amin-ab786a1b0/)
+
+👤 **Heber Lazo**
+
+- [@lazo_heber](https://www.linkedin.com/in/heber-lazo-benza-523266133/) [<img align="left" alt="lazo_heber | Twitter" width="22px" src="https://cdn.jsdelivr.net/npm/simple-icons@v3/icons/twitter.svg" />][twitter]
+
+- [@lazo_heber](https://twitter.com/lazo_heber) [<img align="left" alt="codeSTACKr | LinkedIn" width="22px" src="https://cdn.jsdelivr.net/npm/simple-icons@v3/icons/linkedin.svg" />][linkedin]
```
Comment (descholar-ceo) [inline README.md]: This link is twitter, not linkedIn, you confused icons

File: README.md [hunk 2]
```diff
@@ -1 +1,51 @@
...
+
+😎 **Abdo Amin**
+
+- GitHub: [@Abdelrhman-Amin](https://github.com/AbdelrhmanAmin)
+- Twitter: [@Abdo-Amin](https://twitter.com/AbdoAmi60489112)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/abdo-amin-ab786a1b0/)
+
+👤 **Heber Lazo**
+
+- [@lazo_heber](https://www.linkedin.com/in/heber-lazo-benza-523266133/) [<img align="left" alt="lazo_heber | Twitter" width="22px" src="https://cdn.jsdelivr.net/npm/simple-icons@v3/icons/twitter.svg" />][twitter]
```
Comment (descholar-ceo) [inline README.md]: This link is linkedin, not twitter, you confused icons

File: README.md [hunk 3]
```diff
@@ -1 +1,51 @@
-# Apple-clone
\ No newline at end of file
+# Apple-clone
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Building with Backgrounds and Gradients
+
+This assignment consists of making a clone of Apple's web page.
```
Comment (descholar-ceo) [inline README.md:7]: Work a bit more on this description

File: index.html [hunk 4]
```diff
@@ -4,7 +4,79 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Document</title>
```
Comment (descholar-ceo) [inline index.html]: [Optional]You can edit this title as well,

**Review by descholar-ceo — CHANGES_REQUESTED** [review]
# Status: Change Required :vertical_traffic_light: :warning: 
## Hello @Wusinho and @AbdelrhmanAmin  :handshake: :handshake: :handshake: 
:+1: This is good to make it this far, your clone looks like the original page and it is really nice, congratulations on the good work you have done so far.
:+1: I liked that you have tried to implemented the requested changes from the previous review, keep it up, and your project is really almost done.
### :warning: However we need to make few changes to mark this project approved, don't feel discouraged because of being asked some changes more and more, that's beacuse this is a learning process, and we are here to help you to become an out-standing developer. See the requested change below :point_down: :point_down: :point_down: 
## Note: All of requested changes which are marked as [Required] are mandatory to be implemented in order to get this project approved, others which are marked as [Optional] are nice to have in your project to make it outstanding

# [Required Changes]
- [x] Fix the stylelint lint errors as it was requested by the previous TSE
- [x] Center the footer as well

| Your clone | Original page |
| --- | --- |
| Screenshot from 2020-11-02 20-01-05 | Screenshot from 2020-11-02 20-05-10|
:point_right: Could you see the difference :question: :question: :question: 

- [x] Implement all changes I suggested in inline comments in your codes

# [Optional changes]
- [x] Your commit messages should be in imperative mode or present tense, you should have used `Fix header background` instead of `Fixed header background`, refer [to this article for more details on how to write a good && professional commit message](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8)

If anything seems to be unclear to you, don't hesitate to contact me for more clarification slack: `@Mugirase Emmanuel` | [LinkedIn](https://linkedin.com/in/mugirase-emmanuel) | [Twitter](https://twitter.com/descholar3)

Reviewed by [Mugirase Emmanuel (descholar)](https://github.com/descholar-ceo)

**Review by descholar-ceo — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:  
## :clap: :clap: :clap: Congratulations @Wusinho and @AbdelrhmanAmin , your project is approved :tada: :tada: :tada: 
You have implemented all minimum requirements for this project to mark it as complete, it doesn't need more changes, you can go ahead and merge all of the changes on base branch,

Don't forget to fill the mandatory project completion form, and see you to next project,

Happy coding :computer: :computer: :computer: 

Re-reviewed by [Mugirase Emmanuel (descholar)](https://github.com/descholar-ceo)

**Pull request merged**